docs/setup/adding-a-git-repository.md

```diff
@@ -229,6 +229,110 @@ The following configuration options are supported:
           enable_creation_date: true
     ```
 
+    !!! note
+        If you are deploying through a CI system, please do the CI configuration as described in the [git-revision-date-localized](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin#note-when-using-build-environments) documentation.
+
+        [With Github Actions](../publishing-your-site.md#with-github-actions):
+
+        === "Material for MkDocs"
```

Comment:
Please don't copy our entire CI setup guide, as we'd now have two places where we have to maintain it. Please just clarify with a sentence that when building in a CI environment, you might need to adjust the setting when fetching the code. That's exactly what @alexvoss mentioned. Thanks.

Reply:
Ok, thanks! I will change it.